### DISCUSSION TITLE
limit users to view some Mail Servers

### DISCUSSION THREAD

**Discussion by lanserosk** [postalserver/postal#2469] [Help with using Postal]
Hello, i need to provide users to someone but i cant let him view All Mail Servers in his organization, how can i  limit a user to view only 1 MailServer in the organization?

extra: exist a command list for postal command line?

**Comment by willpower232** [comment]
I don't think that is possible, it looks like there is an `all_servers` boolean in the database but I can't find if that is actually used.

**Reply by lanserosk** [reply]
all_servers is an attribute of 'users' table?,  thank you for reply!
